### ISSUE TITLE
Captured image folder not exist

### ISSUE THREAD

**Issue by pithermal** [tomshaffner/PiThermalCam#28]
Hi
Firstly thanks for this excellent document on setting up the thermal camera 
It all works for me except that the saved images/screenshots do not appear to be saved.
The Python console says that it is saved to a directory in /home/pi, yet when I look for this directory does not exist.
Where would the jpg be stored?
Or
How can I set up a specific directory to store them, eg a usb drive?

thanks
Pithermal

**Comment by tomshaffner** [comment]
Strange; I don't remember if the camera creates that directly. If you create the folder manually do the images save properly then?

Sent from an Android device, please excuse my brevity.

Diese Nachricht wurde von meinem Android Mobiltelefon gesendet, bitte meine Kürze entschuldigen. 
 
  Am So., Jan. 8, 2023 at 6:58 schrieb ***@***.***>:   


Hi
Firstly thanks for this excellent document on setting up the thermal camera
It all works for me except that the saved images/screenshots do not appear to be saved.
The Python console says that it is saved to a directory in /home/pi, yet when I look for this directory does not exist.
Where would the jpg be stored?
Or
How can I set up a specific directory to store them, eg a usb drive?

thanks
Pithermal

**Comment by pithermal** [comment]
Hi
Thanks

I manually created the directory structure in /home/pi and now whenever  I save an image it now appears in the correct directory

Who would have thought it would be that simple!

Thanks

pithermal

**Comment by tomshaffner** [comment]
Agreed, but this is a simple fix in the code too; we should create an issue on GitHub for it to get fixed next time I or someone else are making changes.

Sent from an Android device, please excuse my brevity.

Diese Nachricht wurde von meinem Android Mobiltelefon gesendet, bitte meine Kürze entschuldigen. 
 
  Am So., Jan. 8, 2023 at 12:28 schrieb ***@***.***>:   


Hi
Thanks

I manually created the directory structure in /home/pi and now whenever I save an image it now appears in the correct directory

Who would have thought it would be that simple!

Thanks

pithermal

**Comment by pithermal** [comment]
HiThanks Should there  be some form of error message if the code tries to write to a directory that did not exist? Thanks , you must have put a lot of work in writing the code and producing your illustrated note, which made it much easier for me to get the thermal camera to work pithermal    Sent from Mail for Windows From: tomshaffnerSent: 08 January 2023 18:58To: tomshaffner/PiThermalCamCc: pithermal; AuthorSubject: Re: [tomshaffner/PiThermalCam] Captured image folder not exist (Issue #28) Agreed, but this is a simple fix in the code too; we should create an issue on GitHub for it to get fixed next time I or someone else are making changes.Sent from an Android device, please excuse my brevity.Diese Nachricht wurde von meinem Android Mobiltelefon gesendet, bitte meine Kürze entschuldigen. Am So., Jan. 8, 2023 at 12:28 schrieb ***@***.***>: HiThanksI manually created the directory structure in /home/pi and now whenever I save an image it now appears in the correct directoryWho would have thought it would be that simple!Thankspithermal—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>
